src/bindings/jna/libelektra4j/src/main/java/org/libelektra/exception/KDBException.java

```diff
@@ -0,0 +1,90 @@
+package org.libelektra.exception;
+
+import org.libelektra.Key;
+import org.libelektra.exception.model.WarningEntry;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Custom KDB exception class being used for I/O errors
```

Comment:
for I/O errors?

Reply:
This documentation was copied from the old Exception class which I patched. I was also wondering why `I/O` errors are meant but thought it will have some special meaning behind it. I can remove it

Reply:
Please do not copy text you do not understand. I do not think that any part of that text is correct for what you implemented here. (It is not "Custom", and not used for I/O errors.)

Reply:
> Please do not copy text you do not understand. I do not think that any part of that text is correct for what you implemented here. (It is not "Custom", and not used for I/O errors.)

Neither was it before then obviously but you usually complain if I delete any comments so I kept them in case the author before me know more than I did. I will remove them

Reply:
Because both is wrong:

1. having wrong comments
2. having no comments

Please properly document your methods according design-by-contract.